apps/cowswap-frontend/src/modules/trade/pure/TradeConfirmation/index.tsx

```diff
@@ -16,11 +16,24 @@ import { ConfirmButton } from './ConfirmButton'
 import { ConfirmWarnings } from './ConfirmWarnings'
 import { QuoteCountdown } from './CountDown'
 import { useIsPriceChanged } from './hooks/useIsPriceChanged'
+import { useWarningStates } from './hooks/useWarningStates'
 import * as styledEl from './styled'
 
 import { NoImpactWarning } from '../../containers/NoImpactWarning'
 import { useTradeConfirmState } from '../../hooks/useTradeConfirmState'
 
+function createHookDetailsElement(appData: string | AppDataInfo | undefined): ReactNode {
```

Comment:
Render function

Reply:
Addressed in https://github.com/cowprotocol/cowswap/pull/6022/commits/735fd166e8605b40c637e9ba330a9138480538df